### PR TITLE
Khushboo work

### PR DESCRIPTION
**Pull request by khushboochauhan107**

### CODE REVIEW

File: KHUSHBOO/Assingments/Assignment.6.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (MKadeer17) [inline KHUSHBOO/Assingments/Assignment.6.html]: Give Title Name Proper.

File: KHUSHBOO/Assingments/Assignment.6.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="stylesheet" href="ass6.css">
+</head>
+<body class="f" class="space">
```
Comment (MKadeer17) [inline KHUSHBOO/Assingments/Assignment.6.html:10]: Little bit indentation

File: KHUSHBOO/Assingments/Assignment.6.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (MKadeer17) [inline KHUSHBOO/Assingments/Assignment.6.html]: Little bit indentation

**Review by priyaughade — APPROVED** [review]

File: KHUSHBOO/Assingments/Assignment6/Assignment.6.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="ass6.css">
+</head>
+<body>
+    <div class="f" class="space">
+    <h1 class="bg">M-10 Intern HTML Assesment</h1>
+    <br>
+    <h2 class="bg2">Personal Details</h2>
```
Comment (MKadeer17) [inline KHUSHBOO/Assingments/Assignment6/Assignment.6.html:14]: use Form tag here
before the h2 tag

File: KHUSHBOO/Assingments/Assignment6/Assignment.6.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    <input type="radio" name="age">Less than 18
+    <input type="radio" name="age" checked>18-25
+    <input type="radio" name="age">26-40
+    <br>
+    <h2 class="bg3">Academic Details</h2>
+    <label for="skillset" class="b">Skill Set:</label>
+    <input type="checkbox" name="skill">HTML
+    <input type="checkbox" name="skill">Javascript
+    <input type="checkbox" name="skill">Java
+    <input type="checkbox" name="skill">Oracle
```
Comment (MKadeer17) [inline KHUSHBOO/Assingments/Assignment6/Assignment.6.html:36]: close form tag after the last input tag

File: KHUSHBOO/Assingments/Assignment.6.html (hunk 1 above)
Comment (khushboochauhan107) [reply KHUSHBOO/Assingments/Assignment.6.html]: ok

File: KHUSHBOO/Assingments/Assignment.6.html (hunk 2 above)
Comment (khushboochauhan107) [reply KHUSHBOO/Assingments/Assignment.6.html:10]: ok

File: KHUSHBOO/Assingments/Assignment6/Assignment.6.html (hunk 5 above)
Comment (khushboochauhan107) [reply KHUSHBOO/Assingments/Assignment6/Assignment.6.html:36]: this one is previous i.e wrong one

File: KHUSHBOO/Assingments/Assignment6/Assignment.6.html (hunk 4 above)
Comment (khushboochauhan107) [reply KHUSHBOO/Assingments/Assignment6/Assignment.6.html:14]: this one is previous i.e wrong one

Comment (MKadeer17) [reply KHUSHBOO/Assingments/Assignment6/Assignment.6.html:14]: no
everything in form tag not h1

**Review by MKadeer17 — APPROVED** [review]
Nice

**Review by MKadeer17 — APPROVED** [review]

**Review by omerana14 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by monicapathania — APPROVED** [review]

**Review by pratikshaghule — APPROVED** [review]

**Pull request merged**